### PR TITLE
support regions table

### PR DESCRIPTION
**Pull request by roneli**

### CODE REVIEW

File: resources/provider.go [hunk 1]
```diff
@@ -20,6 +20,7 @@ func Provider() *provider.Provider {
 		ErrorClassifier: client.ErrorClassifier,
 		Migrations:      awsMigrations,
 		ResourceMap: map[string]*schema.Table{
+			"regions":                               Regions(),
```
Comment (bbernays) [inline resources/provider.go:23]: should this be `aws.regions`?

**Review by bbernays — COMMENTED** [review]
LGTM. What about e2e tests?

File: resources/provider.go (hunk 1 above)
Comment (roneli) [reply resources/provider.go:23]: ye I can name it that

**Review by bbernays — APPROVED** [review]
LGTM... Did you decide not to change the name of the resource from `regions` to `aws.regions`?

**Comment by roneli** [comment]
Closed #263

**Pull request merged**